html_css/js/storage.js

```diff
@@ -0,0 +1,68 @@
+export default class Storage {
+  constructor() {
+  }
+
+  getSliderData() {
+    this.initStorage();
+
+    return this.data = JSON.parse(localStorage.getItem('localStorageSliderData'));
+  }
+
+  initStorage() {
+    this.checkLocalStorage();
+    this.setFormToLocalStorage();
+    this.sendButtonHandler();
+  }
+
+  checkLocalStorage() {
+    this.formUserName = JSON.parse(localStorage.getItem('username'))?.trim();
+    this.formTelephone = JSON.parse(localStorage.getItem('telephone'))?.trim();
+    this.formEmail = JSON.parse(localStorage.getItem('email'))?.trim();
+
+    if (this.formUserName || this.formTelephone || this.formEmail) {
+      this.renderFormData();
+    }
+  }
+
+  renderFormData() {
+    this.formUserName ? document.querySelector('#username').value = this.formUserName : null;
```

Comment:
Better to do opposite, like this:

document.querySelector('#username').value = this.formUserName
    ? this.formUserName
    : null;

Here and in similar code below

Reply:
Done